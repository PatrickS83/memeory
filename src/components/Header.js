import React from 'react';

const Header = () => (
  <section className="hero is-info">
    <div className="hero-body">
      <div className="container">
        <h1 className="title">
          Memeory
      </h1>
        <h2 className="subtitle">
          Like memory. But fun.
      </h2>
      </div>
    </div>
  </section>
);

export default Header;
