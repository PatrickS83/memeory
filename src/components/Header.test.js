import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('<Header/>', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.length).toEqual(1);
  });

  it('renders the Header as saved in snapshot', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});