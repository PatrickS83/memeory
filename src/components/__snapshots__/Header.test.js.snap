// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> renders the Header as saved in snapshot 1`] = `
<section
  className="hero is-info"
>
  <div
    className="hero-body"
  >
    <div
      className="container"
    >
      <h1
        className="title"
      >
        Memeory
      </h1>
      <h2
        className="subtitle"
      >
        Like memory. But fun.
      </h2>
    </div>
  </div>
</section>
`;
